doc/pius.1

```diff
@@ -47,6 +47,8 @@ Hostname of SMTP server. [default: \fIlocalhost\fP]
 Use the pexpect module for signing and drop to the gpg shell for entering the passphrase. [default: false]
 .IP "\fB\-I\fP, \fB\-\-import\fP"
 Also import the unsigned keys from the keyring into the default keyring. Ignored if \fB\-r\fP is not specified, or if it's the same as the default keyring.
+.IP "\fB\-L\fP, \fB\-\-local\-mail\-dir\fP"
```

Comment:
We should probably call this `--save-to-mail-dir` or something

Reply:
Changed to --save-to-mail-dir